vision/snippets/quickstart/quickstart.py

```diff
@@ -42,11 +45,10 @@ def run_quickstart():
     response = client.label_detection(image=image)
     labels = response.label_annotations
 
-    print('Labels:')
-    for label in labels:
-        print(label.description)
-    # [END vision_quickstart]
+    return labels
```

Comment:
suggestion: As this is a quickstart I suggest leaving the print statements to show how to iterate through labels plus returning them for the test purposes.

Reply:
Yup. Makes sense. Added it back.